### PR TITLE
#26912 Glossary: Baseline

### PR DESCRIPTION
**Pull request by estelle**
suggested solution for https://github.com/mdn/content/issues/26912

### CODE REVIEW

**Comment by hamishwillee** [comment]
@ddbeck Is this something you would like to review? I see from your comments on https://github.com/mdn/content/issues/26912#issuecomment-2186508455 that you have opinions, while I do not.

**Comment by Rumyra** [comment]
Thanks for the pr @estelle - I'd like to keep this drafted while we discuss where the banners should link to (if the glossary page doesn't explain what baseline is that's a larger discussion than we anticipated) - we have a meeting on Thursday and I should be able to confirm more then

**Comment by ddbeck** [comment]
@hamishwillee @estelle Yes, I'd like to have a look at this when it's not a draft. Thank you!

**Comment by estelle** [comment]
@ddbeck @Rumyra This is just a draft for discussion; figured this was the best way to review provide a issue resolution suggestion for feedback

**Comment by estelle** [comment]
@Rumyra @ddbeck I pull this out of "draft" and switched to "ready for review" as no one expressed any concerns. Feel free to review, or put back in draft and discuss. Thanks.

File: files/en-us/glossary/baseline/compatibility/index.md [hunk 1]
```diff
@@ -43,23 +16,21 @@ Baseline tracks availability with the following browsers:
 - Mozilla Firefox (Android)
 - Mozilla Firefox (desktop)
 
-## Extra considerations
+Baseline is a summary guide to browser support, but does address feature accessibility, usability, performance, security, or other concerns.
```
Comment (ddbeck) [inline files/en-us/glossary/baseline/compatibility/index.md:19]: I'd really like to preserve the original text more closely. It's the most picked over, negotiated bit of the original text. But in the spirit of brevity, my take on this:

```suggestion
Baseline is a summary of browser support. It is not a substitute for accessibility, usability, performance, security, or other testing. Baseline may not tell if you if a feature works with:

- Older devices and browser releases
- Browsers not covered by the Baseline definition, such as operating system web views
- Assistive technology, such as screen readers.
```

File: files/en-us/glossary/baseline/compatibility/index.md [hunk 2]
```diff
@@ -43,23 +16,21 @@ Baseline tracks availability with the following browsers:
...
+![Green widget with the checkmark: Baseline, widely available. Four browsers logos, all with checkmarks.](high.png)
+
+Features listed as **widely available** have a consistent history of browser support for at least 2.5 years.
 
-- Older devices and browser releases
-- Browsers not covered by the Baseline browser set, such as operating system web views
-- Assistive technology, such as screen readers
+![Blue widget with the checkmark: Baseline 2022, newly available. Four browsers' logos, all with checkmarks.](limited.png)
 
-If Baseline doesn't cover your situation, you can find more information in the browser compatibility tables on MDN, [caniuse.com](https://caniuse.com/), and [a11ysupport.io](https://a11ysupport.io/).
```
Comment (ddbeck) [inline files/en-us/glossary/baseline/compatibility/index.md:57]: I don't have a direct suggestion here, except to say that acknowledging that Baseline is _not_ the only source of information on this stuff, particularly on the non-browser accessibility support, was essential to previous edits to this page.

File: files/en-us/glossary/baseline/compatibility/index.md [hunk 3]
```diff
@@ -43,23 +16,21 @@ Baseline tracks availability with the following browsers:
...
+Features listed as **newly available** works in all stable browser versions, but may not work with older browsers and devices.
```
Comment (ddbeck) [inline files/en-us/glossary/baseline/compatibility/index.md:29]:
```suggestion
Features listed as **newly available** work in at least the latest stable version of each of the Baseline browsers, but may not work with older browsers and devices.
```

"at least" is key here—features spend 2.5 years in this state. Also, I want to be careful not to imply that Baseline is "all" browsers. Its a specific set that gets reviewed periodically.

File: files/en-us/glossary/baseline/compatibility/index.md [hunk 4]
```diff
@@ -43,23 +16,21 @@ Baseline tracks availability with the following browsers:
...
+Baseline is a summary guide to browser support, but does address feature accessibility, usability, performance, security, or other concerns.
 
-Baseline is a summary guide to browser support to help you make decisions before committing to use a feature in your projects.
-Baseline is not a substitute for testing your application for accessibility, usability, performance, security, or other concerns.
+## Baseline badges
 
-Baseline may not tell you whether a feature works with:
+![Green widget with the checkmark: Baseline, widely available. Four browsers logos, all with checkmarks.](high.png)
+
+Features listed as **widely available** have a consistent history of browser support for at least 2.5 years.
```
Comment (ddbeck) [inline files/en-us/glossary/baseline/compatibility/index.md:25]:
```suggestion
Features listed as **widely available** have a consistent history of support in each of the Baseline browsers for at least 2.5 years.
```

Not just 2.5 years in general, but a minimum of 2.5 years across the whole set of browsers.

**Review by ddbeck — COMMENTED** [review]
Sorry for my tardiness in reviewing. Thank you for considering this, Estelle!

**Comment by estelle** [comment]
@ddbeck i committed your suggestions and added caniuse and a11yio to the see also section

**Review by ddbeck — COMMENTED** [review]
This is good enough for me. Thank you!

**Comment by Josh-Cena** [comment]
@Rumyra Could you review this?

**Review by Rumyra — APPROVED** [review]

**Comment by Rumyra** [comment]
Thanks both - as @ddbeck is happy so am I - merging

**Pull request merged**